src/components/organisms/cards/cards.html

```diff
@@ -0,0 +1,40 @@
+<div class="card-group">
```

Comment:
```suggestion
<div class="card-group container">
```
containerはこんな感じで親の要素に設定するので問題ないと思います。